src/utils/collection/index.ts

```diff
@@ -195,7 +195,20 @@ export const isSystemCollection = (type: CollectionSummaryType) => {
 export const shouldShowOptions = (type: CollectionSummaryType) => {
     return !OPTIONS_NOT_HAVING_COLLECTION_TYPES.has(type);
 };
-
+export const showTrashQuickOption = (type: CollectionSummaryType) => {
+    return type === CollectionSummaryType.trash;
+};
+export const showDownloadQuickOption = (type: CollectionSummaryType) => {
+    return !(type === CollectionSummaryType.trash);
```

Comment:
`type !== CollectionSummaryType.trash ` 
simpler?

Reply:
also as suggested in the last comment... let's say which to show for... rather than for which to not show